src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -138,7 +139,9 @@ def create(dataset, session_id, target, features=None, prediction_window=100,
     from .._mps_utils import (use_mps as _use_mps,
                               mps_device_name as _mps_device_name,
                               ac_weights_mps_to_mxnet as _ac_weights_mps_to_mxnet)
-
+    from ._tf_model_architecture import ActivityTensorFlowModel, _fit_model_tf
```

Comment:
Won't this always trigger an import of TensorFlow? We should only do this if the user has requested TensorFlow

It's important that this PR have no significant effect in master when `use_tensorflow` is false